extdata/lang.dat

```diff
@@ -66,4 +66,5 @@
 .r : R
 .toml: toml
 .swf: swift
-.sh : bash
\ No newline at end of file
+.sh : bash
+.pro : prolog
```

Comment:
add .pl besides .pro for associating that extension to prolog too.

Reply:
.pl is currently associated with perl.  When I added it for prolog as well running `cnh this test.pl` generated a perl file instead of prolog.  I think perl is much more common than prolog so it's probably better to leave it with perl.  If there's something I missed with how to support multiple languages for an extension I'm happy to add it, though.

Reply:
I think it's important to add the whole extensions here in lang.dat file. Currently the system is unable to support multiple languages for a common extension but in future developments It would be a good starting poing lang.dat to know how to migrate that data to a database. Btw, the issue #9  refers to this topic.

Reply:
I had a busy weekend away from the computer.  I will add the suffix and make sure nothing breaks this evening.